packages/web/src/services/audius-backend/BuyAudio.ts

```diff
@@ -52,7 +52,7 @@ export const getRootAccountRentExemptionMinimum = async () => {
     (await connection.getMinimumBalanceForRentExemption(
       ROOT_ACCOUNT_SIZE,
       'processed'
-    )) + 5000 // Temporary fix to pad such that we can always transfer if there's leftovers
+    )) + 10_000 // Temporary fix to pad such that we can always transfer if there's leftovers
```

Comment:
just a headsup that _ doesn't work in react-native sadly. so we are avoiding it just in case